packages/services/api/src/modules/alerts/resolvers.ts

```diff
@@ -217,4 +217,13 @@ export const resolvers: AlertsModule.Resolvers = {
       return channel.webhookEndpoint!;
     },
   },
+  TeamsWebhookChannel: {
+    __isTypeOf(channel) {
+      return channel.type === 'MSTEAMS_WEBHOOK';
+    },
+    endpoint(channel) {
+      // @ts-expect-error TODO not sure where this type is coming from
```

Comment:
You need to configure the mapper type for `TeamsWebhookChannel` within the `codegen.mts` file. See  ho it is solved for the `AlertWebhookChannel` object type.

Reply:
added now